irekon/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "irekon"
+version = "0.1.0"
+edition = "2024"
```

Comment:
Missing `license`.

(And this commit needs to be squashed with the previous ones, when we introduce `irekon` and when adding the different dependencies.)